api-report/container-definitions.api.md

```diff
@@ -51,15 +55,7 @@ export namespace ConnectionState {
 // @public
 export type ConnectionState = ConnectionState.Disconnected | ConnectionState.EstablishingConnection | ConnectionState.CatchingUp | ConnectionState.Connected;
 
-// @public
-export enum ContainerErrorType {
-    clientSessionExpiredError = "clientSessionExpiredError",
-    dataCorruptionError = "dataCorruptionError",
-    dataProcessingError = "dataProcessingError",
-    genericError = "genericError",
-    throttlingError = "throttlingError",
-    usageError = "usageError"
-}
+export { ContainerErrorType }
```

Comment:
i think you should keep the enum in this package, and just deprecate it, the const object isn't 1:1 compatible

Reply:
Got it. So in core-interfaces, I would turn it from a const into an enum. And simply deprecate the one in this package.